src/components/Dashboard/DashboardGrid.jsx

```diff
@@ -184,7 +184,15 @@ export const findLayoutOrGenerate = (layouts, cards, supportedLayouts) => {
 
     const layoutWithResizableItems = layout.map((cardFromLayout) => {
       const matchingCard = find(cards, { id: cardFromLayout.i });
-      return { ...cardFromLayout, isResizable: matchingCard.isResizable };
+      return {
+        ...cardFromLayout,
+        // only add resizable attribute if it exists
+        ...(matchingCard.isResizable
```

Comment:
Why was this needed? If using `undefined` was problematic we could perhaps use `isResizable: !!matchingCard.isResizable` instead of conditionally excluding the property.

Reply:
Its not a problem. I just saw it getting added in calls. I'll add it back in